### ISSUE TITLE
YDK netconf read fails when <data> tag has namespace prefix

### ISSUE THREAD

**Issue by abhikeshav** [CiscoDevNet/ydk-gen#799]
Issue tracker is ONLY used for reporting bugs. Please use the [YDK Community](https://communities.cisco.com/community/developer/ydk) for any support issues.

## Expected Behavior
`data` tag with namespace prefix should work

## Current Behavior

Below reply from device causes `ydk.errors.YServiceProviderError: Can't find data tag in reply sent by device` error
```
<nc:rpc-reply xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" xmlns:junos="http://xml.juniper.net/junos/15.1X49/junos" xmlns="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="1">
  <nc:data>
...
</nc:data>
</nc:rpc-reply>
```

**Comment by abhikeshav** [comment]
Also fails if the `rpc-reply` for `get` or `get-config` contains `<data>` tag within another `<data>` tag

**Comment by ygorelik** [comment]
Opened IOS XR bug CSCvn36121 (Cisco-IOS-XR-bundlemgr-oper Yang model conflicts with IETF standard). The XR development team agreed and took it to work. Therefore closing this issue.

**Comment by ygorelik** [comment]
The <data> tag within another <data> tag issue has been fixed in YDK-0.8.1. See issue [851 ](https://github.com/CiscoDevNet/ydk-gen/issues/851) for details

**Comment by ygorelik** [comment]
C++ Test
```
namespace ydk {
  namespace path {
    std::string get_netconf_output(const string & reply);
  }
}

TEST_CASE( "decode_rpc_with_nc_prefix" )
{
    auto rpc_reply = R"(
<nc:rpc-reply xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" xmlns:junos="http://xml.juniper.net/junos/15.1X49/junos" xmlns="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="1">
<nc:data>
<interfaces xmlns="http://openconfig.net/yang/interfaces">
  <interface>
    <name>GigabitEthernet0/0/0/2</name>
    <config>
      <type xmlns:ianaift="urn:ietf:params:xml:ns:yang:iana-if-type">ianaift:ethernetCsmacd</type>
      <name>GigabitEthernet0/0/0/2</name>
    </config>
  </interface>
</interfaces>
</nc:data>
</nc:rpc-reply>
)";
    auto reply_xml = ydk::path::get_netconf_output(rpc_reply);
    REQUIRE(!reply_xml.empty());

    auto rpc_reply_extracted = R"(<interfaces xmlns="http://openconfig.net/yang/interfaces">
  <interface>
    <name>GigabitEthernet0/0/0/2</name>
    <config>
      <type xmlns:ianaift="urn:ietf:params:xml:ns:yang:iana-if-type">ianaift:ethernetCsmacd</type>
      <name>GigabitEthernet0/0/0/2</name>
    </config>
  </interface>
</interfaces>)";
    REQUIRE(reply_xml == rpc_reply_extracted);
}
```
